### ISSUE TITLE
Pinned messages

### ISSUE THREAD

**Issue by officialdakari** [SchildiChat/SchildiChat-android#202]
**Is your feature request related to a problem? Please describe.**
No

**Describe the solution you'd like**
Add pinned messages like on SchildiChat web version

**Describe alternatives you've considered**
Usage of pinned messages in fluffychat

**Additional context**
none

**Comment by SpiritCroc** [comment]
Maybe possible to cherry-pick from https://github.com/vector-im/element-android/pull/7762

Update: picked to https://github.com/SchildiChat/SchildiChat-android/commits/pinned-messages, but doesn't seem to work right now to either load pinned messages, or to pin messages. If anybody wants to have a look

**Comment by grvn-ht** [comment]
Hey @SpiritCroc any news on this issue?

**Comment by SpiritCroc** [comment]
No, all work focuses on the [SchildiChat Android rewrite](https://github.com/SchildiChat/schildichat-android-next), so I will not work on this.
